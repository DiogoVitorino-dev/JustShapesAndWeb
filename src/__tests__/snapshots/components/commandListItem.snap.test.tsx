import { render } from "@testing-library/react-native";

import CommandListItem from "@/components/settings/keyboard/commandListItem";

describe("Command List Item - Snapshot component test", () => {
  it("should render correctly", () => {
    const tree = render(<CommandListItem command="UP" />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
