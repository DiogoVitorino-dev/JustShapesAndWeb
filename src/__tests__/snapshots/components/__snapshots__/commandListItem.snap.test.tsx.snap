// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Command List Item - Snapshot component test should render correctly 1`] = `
<View
  style={
    [
      undefined,
      {
        "flexDirection": "row",
      },
    ]
  }
>
  <View
    style={
      [
        undefined,
        [
          {
            "borderColor": "rgb(179, 179, 179)",
            "borderWidth": 0.5,
            "flex": 2,
            "justifyContent": "center",
            "minHeight": 100,
            "minWidth": 100,
          },
          {
            "flex": 1,
          },
        ],
      ]
    }
  >
    <Text
      adjustsFontSizeToFit={true}
      numberOfLines={1}
      style={
        [
          {
            "color": "rgb(245, 245, 245)",
            "fontSize": 22,
          },
          [
            {
              "fontWeight": "bold",
              "textAlign": "center",
            },
            {
              "fontFamily": "Manjari",
              "marginBottom": -3,
            },
          ],
        ]
      }
    >
      UP
    </Text>
  </View>
  <View
    accessibilityState={
      {
        "busy": undefined,
        "checked": undefined,
        "disabled": undefined,
        "expanded": undefined,
        "selected": undefined,
      }
    }
    accessibilityValue={
      {
        "max": undefined,
        "min": undefined,
        "now": undefined,
        "text": undefined,
      }
    }
    accessible={true}
    collapsable={false}
    focusable={true}
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      {
        "borderColor": "rgb(179, 179, 179)",
        "borderWidth": 0.5,
        "flex": 2,
        "justifyContent": "center",
        "minHeight": 100,
        "minWidth": 100,
      }
    }
  >
    <Text
      style={
        [
          {
            "color": "rgb(245, 245, 245)",
            "fontSize": 22,
          },
          [
            [
              {
                "opacity": 1,
              },
              {
                "fontWeight": "bold",
                "textAlign": "center",
              },
            ],
            {
              "fontFamily": "Manjari",
              "marginBottom": -3,
            },
          ],
        ]
      }
    />
  </View>
  <View
    accessibilityState={
      {
        "busy": undefined,
        "checked": undefined,
        "disabled": undefined,
        "expanded": undefined,
        "selected": undefined,
      }
    }
    accessibilityValue={
      {
        "max": undefined,
        "min": undefined,
        "now": undefined,
        "text": undefined,
      }
    }
    accessible={true}
    collapsable={false}
    focusable={true}
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      {
        "borderColor": "rgb(179, 179, 179)",
        "borderWidth": 0.5,
        "flex": 2,
        "justifyContent": "center",
        "minHeight": 100,
        "minWidth": 100,
      }
    }
  >
    <Text
      style={
        [
          {
            "color": "rgb(245, 245, 245)",
            "fontSize": 22,
          },
          [
            [
              {
                "opacity": 1,
              },
              {
                "fontWeight": "bold",
                "textAlign": "center",
              },
            ],
            {
              "fontFamily": "Manjari",
              "marginBottom": -3,
            },
          ],
        ]
      }
    />
  </View>
</View>
`;
